src/backend/utils.ts

```diff
@@ -899,6 +899,26 @@ async function ContinueWithFoundWine(
   selectedWine: string,
   foundWine: string
 ): Promise<{ response: number }> {
+  const isGPTK = selectedWine.toLowerCase().includes('toolkit')
```

Comment:
The check here for toolkit only is intentional. If I want to avoid all toolkits, even from third-party like Whisky, I would need to add more stuff. But since those other GPTK might be modified and even run on older versions, I think we should try.
this is definitely something that only pretty advanced macOS users would notice anyway and they probably knows their ways around it.